networkx/conftest.py

```diff
@@ -54,16 +54,22 @@ def set_warnings():
         message="This will return a generator in 3.0*",
     )
     warnings.filterwarnings(
-        "ignore", category=DeprecationWarning, message="betweenness_centrality_source*"
+        "ignore", category=DeprecationWarning, message="betweenness_centrality_source*",
```

Comment:
You are adding a comma at the end of the line, which is why black is complaining.  You should just leave lines as the are.

Reply:
Ah I see - I missed the trailing comma so `black` was automatically reverting to the one-arg-per-line format. Should be fixed now.